.changeset/neat-lights-worry.md

```diff
@@ -0,0 +1,22 @@
+---
+"@fluidframework/datastore-definitions": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
+---
+---
+"section": deprecation
+---
+
+The function `process` has been deprecated on the following interface - `IFluidDataStoreChannel`. It has also been deprecated on the following mock test class - `MockFluidDataStoreRuntime`.
+
+A new function `processMessages` has been added in its place which will be called to process multiple messages instead of a single one. This is part of a feature called "Op bunching" where contiguous ops of a given type and to a given data store / DDS are bunched and sent together for processing.
+
+---
+"section": other
+---
```

Comment:
@tylerbutler are two sections allowed in a single changeset? I don't know if the tool will handle it correctly or if this needs to be split into a second file.

Reply:
I wasn't sure about this one either. But I could not find any documentation or example of this.

Reply:
Create two changesets if you have two different changes to describe.

Reply:
Updated it to two changesets